modules/system/classes/CombineAssets.php

```diff
@@ -464,7 +464,7 @@ protected function prepareCombiner(array $assets, $rewritePath = null)
      */
     protected function setHashOnCombinerFilters($hash)
     {
-        $allFilters = call_user_func_array('array_merge', $this->getFilters());
+        $allFilters = array_merge(...array_values($this->getFilters()));
```

Comment:
In php8 named parameters were introduced and now it is required to match called method parameter name when setting parameters by array destructing or call_user_func_array() etc.